python-backend/app/api/document_manager/resources/document_manager.py

```diff
@@ -0,0 +1,85 @@
+import decimal
+import uuid
+import os
+import re
+from datetime import datetime
+from werkzeug.datastructures import FileStorage
+from werkzeug import exceptions
+from flask import request, current_app
+from flask_restplus import Resource, reqparse
+from flask_uploads import UploadNotAllowed
+
+from ..models.document_manager import DocumentManager
+from app.extensions import jwt, api, documents
+from ...utils.resources_mixins import UserMixin, ErrorMixin
+
+
+class DocumentManagerResource(Resource, UserMixin, ErrorMixin):
+    parser = reqparse.RequestParser()
+    
+    @jwt.requires_roles(["mds-mine-create"])
+    def post(self):
+
+        self.parser.add_argument('file', type=FileStorage, location='files', action='append')
+        self.parser.add_argument('folder', type=str, help='mine to the new document on')
```

Comment:
I don't understand this help message. Perhaps it makes sense in the context it would be shown in?